src/features/pastelDB/pastelDBLib.ts

```diff
@@ -63,23 +96,463 @@ export const saveDataToLocalSqlite = async (db: Database): Promise<void> => {
 }
 
 export const createTables = async (db: Database): Promise<void> => {
-  db.exec(create_statisticinfo)
-  db.exec(create_networkinfo)
-  db.exec(create_nettotals)
-  db.exec(create_mempoolinfo)
-  db.exec(create_rawmempoolinfo)
-  db.exec(create_mininginfo)
-  db.exec(create_block)
-  db.exec(create_rawtransaction)
-  db.exec(create_transaction)
-  db.exec(create_txoutsetinfo)
-  db.exec(create_chaintips)
-  db.exec(create_blocksubsidy)
-  db.exec(create_walletinfo)
-  db.exec(create_listreceivedbyaddress)
-  db.exec(create_listtransactions)
-  db.exec(create_listunspent)
-  db.exec(create_totalbalance)
-  db.exec(create_listaddresses)
+  db.exec(createStatisticinfo)
+  db.exec(createNetworkinfo)
+  db.exec(createNettotals)
+  db.exec(createMempoolinfo)
+  db.exec(createRawmempoolinfo)
+  db.exec(createMininginfo)
+  db.exec(createBlock)
+  db.exec(createRawtransaction)
+  db.exec(createTransaction)
+  db.exec(createTxoutsetinfo)
+  db.exec(createChaintips)
+  db.exec(createBlocksubsidy)
+  db.exec(createWalletinfo)
+  db.exec(createListreceivedbyaddress)
+  db.exec(createListtransactions)
+  db.exec(createListunspent)
+  db.exec(createTotalbalance)
+  db.exec(createListaddresses)
   return
 }
+
+export const validateDataFromDB = (
+  pastelDB: Database,
+  tableName: string,
+  transactionid: string,
+  time: number,
+): boolean => {
+  const sqlText = `SELECT id FROM ${tableName} WHERE transactionid=$tid AND time=$time`
```

Comment:
Let's move this to queries (and all other SQL queries to queries.ts file).

Also, we need to add some table name validations if `$tableName` is not available as a query parameter. The reason I'm asking is to guard us from SQL injections: imagine I provide this string instead of `$tablename`: `(DROP TABLE blockinfo);`. 

Example of validations:
```
const tableNames = {
  blockinfo: true,
  transaction_tbl: true,
  // ... other tables
}
if (tableNames[tableName] !== true) {
  throw new Error(`pastelID validateDataFromDB error: table name is invalid`)
}
```

Reply:
- Good comments, updated.